### PR TITLE
Refresh

### PR DESCRIPTION
**Pull request by belucid**
https://trello.com/c/hochV0ZM

- Update storage API to treat topics as secondary category of entries
- Reset the DB migrations to current schema
- Some minimal work on updates -> stories
- Move repo (open-company-api -> open-company-storage)
- Move the DB name (open_company -> open_company_storage)
- Move port from 3000 to 3001
- Flatten board -> topic -> entry hierarchy to board -> entry
- Make topic an optional aspect of entries
- Provide comments and interactions data in entries at board level
- Remove stale sample data, update remaining sample data

Dependent interaction PR: https://github.com/open-company/open-company-interaction/pull/6

Dependent oc.lib PR: https://github.com/open-company/open-company-lib/pull/15

Dependent infra PR: https://github.com/belucid/open-company-infrastructure/pull/41

To test:

- Reset DBs
- Import all remaining sample data
- Use the latest web branch w/ these set of PRs
- Review the code
- Merge
- Deploy
- Dress up like a gladiator and have mock combat with your dogs

### CODE REVIEW

**Comment by bago2k4** [comment]
@belucid eastwood is failing here. i'll do my review just the same then check the fixing commit

File: src/oc/storage/api/orgs.clj [hunk 1]
```diff
@@ -141,7 +140,7 @@
                              authors (:authors org)
                              author-reps (map #(org-rep/render-author-for-collection org %) authors)
                              update-count (if user
-                                            (count (update-res/list-updates-by-author conn org-id user-id))
+                                            0 ;(count (update-res/list-updates-by-author conn org-id user-id))
                                             0)]
```
Comment (bago2k4) [inline src/oc/storage/api/orgs.clj]: What are the lines above for?

Comment (belucid) [inline src/oc/storage/api/orgs.clj]: This is old code that is counting how many `stakeholder-updates` there were and putting the count on the link. We used to use this info in the web UI. I'll remove it.

**Comment by belucid** [comment]
@bago2k4 ready for you.

**Pull request merged**